src/css/blocks/page.css

```diff
@@ -0,0 +1,11 @@
+.page__body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
body doesn't have default padding
```suggestion
body {
  margin: 0;
}
```